zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -1063,8 +1063,8 @@ private void ensureMutualExclusivity(
     /**
      * Update the about page to the entered value, and show a success message.
      */
-    @Restrict("#{s:hasPermission(projectHome.instance, 'update')}")
     public void updateAboutPage() {
+        identity.checkPermission(instance, "update");
```

Comment:
That (checkPermission in `EntityHome.update()`) sounds like a good refactoring, but for now I'd like to keep it to a dead simple translation where possible.